### PR TITLE
Added banner for old reference doc

### PR DESCRIPTION
**Pull request by alanvww**
Hello @MOQN @QuinnHe , this is the test version of the banner with dismiss button and responsive for mobile. The style is temporary. Let me know what you think!

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -51,6 +51,13 @@
 
   <div id="app">Loading...</div>
 
+    <div id="banner" style="display: none;">
+      <div id="banner-content">
+        <a href="https://archive-docs.ml5js.org/#/" target="_blank">💜 Looking for the old ml5.js reference website? Find it
+          here!</a>
```
Comment (shiffman) [inline docs/index.html:57]: Some tiny language tweaks and maybe take the heart out of the link? 

```suggestion
        💜 <a href="https://archive-docs.ml5js.org/#/" target="_blank">Looking for the old ml5.js reference (v0.12.2) Find it 
        here!</a>
```

**Review by shiffman — COMMENTED** [review]
This is great, thank you @alanvww! Perhaps we should put something on our "calendar" to remove these banners in 3-6 months? (Assuming things are going smoothly with 1.x?)

**Comment by MOQN** [comment]
@alanvww Thank you so much! It looks great to me!! (Also with the edition by Dan)

**Comment by MOQN** [comment]
@shiffman Thank you for the "calendar" idea. How about we maintain the banners in Fall 2024 and remove them during the winter break? (We keep approx. 5 months)

**Comment by alanvww** [comment]
@MOQN @shiffman Thanks for the reviews! I have already committed the edited version. So if the preview deployment looks good, this PR is ready for merge :D 

For the banner removal timeline, I think either way will work, starting with 3 months is the best bet. I will keep that in mind.

**Pull request merged**